wgpu/backends/wgpu_native/_helpers.py

```diff
@@ -234,49 +233,63 @@ class WgpuAwaitable:
     truely async manner, as well as to support a synchronous version of them.
     """
 
-    def __init__(self, title, result, callback, poll_function, finalizer, timeout=5.0):
+    SLEEP_TIME = 0.005  # 5ms
```

Comment:
This should really be 0, we want to resolve the awaitables as fast as possible

Reply:
I think so too. If a user uses `buffer.map()` on every draw, than this wait-time hits performance on each frame. There's no way to come up with a "reasonable" timeout, so I think 0 makes the most sense. That way at least control is given back to the loop / other threads.

Reply:
I have changed it to 0.  Sorry I didn't see that comment.  Github usually lets me know.